mesh/mesh.cpp

```diff
@@ -4508,10 +4508,10 @@ void Mesh::MakeSimplicial_(const Mesh &orig_mesh, int *vglobal)
 
          // Rotate the vertices of the hex so that the smallest vertex index is
          // in the first place
-         int irot = find_min(vg, nv_hex);
+         int imin = find_min(vg, nv_hex);
          for (int iv=0; iv<nv_hex; ++iv)
          {
-            int jv = hex_rot[iv + irot*nv_hex];
+            int jv = hex_rot[iv + imin*nv_hex];
             vg[iv] = v[jv];
          }
```

Comment:
It's better to keep `irot` here and change the index in the `for` loop instead.

Reply:
Will do